src/makeCalculator.js

```diff
@@ -4,7 +4,29 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (a, b) => a + b,
+
+    subtract: (a, b) => a - b,
+
+    multiply: (a, b) => a * b,
+
+    divide: (a, b) => a / b,
```

Comment:
The `divide` method should update the `result` property. Additionally, it should handle division by zero to prevent errors. Consider adding a check for `b === 0` and handling it appropriately.